packages/eas-build-job/src/errors.ts

```diff
@@ -124,8 +125,8 @@ export class IncompatiblePodsManagedWorkflowError extends UserError {
 You are seeing this error because either:
   - Versions in the Podfile.lock cached by EAS do not match required values for some of the libraries, it can be triggered when upgrading Expo SDK or any other library with native code. To fix that ${
     usingDefaultCacheConfig
-      ? 'add the "cache.key" field (it can be set to any value) in eas.json to invalidate the cache.'
-      : 'update value of the "cache.key" field in eas.json to invalidate the cache.'
+      ? 'add the "cache.key" field (it can be set to any value) in the build profile in eas.json to invalidate the cache.'
```

Comment:
unrelated, but i saw recently someone that tried add those keys in root of eas.json